Gemfile

```diff
@@ -8,6 +8,7 @@ group :style do
   gem 'foodcritic', '~> 16.2.0'
   gem 'rake', '~> 13.0.1'
   gem 'rubocop', '~> 0.80.1'
+  gem 'rubocop-gitlab-security'
```

Comment:
Do we need to pin to a version? I assume not because we would want to get the most updated rubocop security linter?

Reply:
It's not a bad idea to pin to the latest version. If the commit from this PR for enabling dependabot PRs works as expected it will remind us to update this whenever new versions are available anyways.